### PR TITLE
UXENG-2120 - Updated docs for new key UI

### PR DESCRIPTION
**Pull request by bnlb**
Resolves the dev-center side of [UXENG-2120](https://algorithmia.atlassian.net/browse/UXENG-2120)

I realize some of the screenshots my change and this could require an update, but all of the language updates are in place.

### CODE REVIEW

**Comment by ryanmillerdev** [comment]
Looking at this now!

File: _pages/model-deployment/allennlp.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Welcome to deploying your <a href="https://allennlp.org/">AllenNLP</a> model on
 
 This guide is designed as an introduction to deploying a AllenNLP model and publishing an algorithm even if you’ve never used Algorithmia before.
 
-Note: this guide uses the web UI to create and deploy your Algorithm. If you prefer a code-only approach to deployment, review [Algorithm Management API]({{site.baseurl}}/algorithm-development/algorithm-management-api) after reading this guide.
+Note: this guide uses the web UI to create and deploy your Algorithm. If you prefer a code-only approach to deployment, review [algorithm management]({{site.baseurl}}/algorithm-development/algorithm-management) after reading this guide.
```
Comment (ryanmillerdev) [inline _pages/model-deployment/allennlp.md]: :bulb: Might update this to "our algorithm management guide".

Comment (ryanmillerdev) [reply _pages/model-deployment/allennlp.md]: Or perhaps capitalize it, just to keep it consistent with the prior link.

File: _pages/algorithm-development/algorithm-management.md [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
+* **Full Access** - if you'd like to allow the key to call all public and private algorithms you have access to, in addition to managing algorithms you own.
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/algorithm-management/full-access.png" alt="Full Access Algorithm Management Option" class="screenshot">
+
+* **Restricted Access** - if you'd like to fine-tune which algorithms can be called. (Just be sure to select the 'Allow this key to manage my algorithms' checkbox.)
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/algorithm-management/restricted-access.png" alt="Restricted Access Algorithm Management Option" class="screenshot">
+
+To see a full list of specific endpoints a key with this permission can call, see the [Python API for Algorithm Management](https://docs.algorithmia.com/?python#algorithm-management-api). For languages other than Python, an OpenAPI Specification is available via the [unofficial openapi docs](https://documenter.getpostman.com/view/6515899/S1a1aoR6?version=latest#4809ce72-446d-40ef-b857-995e7b2949de) or a [raw openapi spec (under /algorithms)](/v1/openapispec).
```
Comment (ryanmillerdev) [inline _pages/algorithm-development/algorithm-management.md:29]: :bulb: I'm a bit hesitant to hype our OpenAPI spec till we have full coverage. My recommendation for this paragraph:

```
To see a full list of specific endpoints a key with this permission can call, see [our API documentation](https://docs.algorithmia.com/?python#algorithm-management-api).
```

Comment (ryanmillerdev) [reply _pages/algorithm-development/algorithm-management.md:29]: Nvm, thought this was a new addition

**Review by ryanmillerdev — APPROVED** [review]
A couple small items–LGTM!

File: _pages/model-deployment/allennlp.md (hunk 1 above)
Comment (bnlb) [reply _pages/model-deployment/allennlp.md]: Updated

**Pull request merged**